shared/types/genesis/proto/gov.proto

```diff
@@ -1,124 +1,232 @@
 syntax = "proto3";
+
 package genesis;
 
 option go_package = "github.com/pokt-network/pocket/shared/types/genesis";
 
 // DISCUSS(drewskey): Explore a more general purpose "feature flag" like approach for this.
 message Params {
+  //@gotags: pokt:"val_type=BIGINT"
```

Comment:
Here I am attaching "metadata" to the fields so that the generated structs have them in the form of `struct tags` that we can parse similarly to what JSON marshaller/unmarshaller do basically everywhere.

Reply:
So does this mean we can use `protoc-go-inject-tag  ` to remove this hack I introduced? https://github.com/pokt-network/pocket/blob/main/shared/types/genesis/validator.go#L10

If so, we should definitely fix my hacks 😅

Reply:
I'll have a look but in my comment above I wasn't referring to anything specific in the codebase but in general what the standard library (I think `encoding/json`) does, ergo: reading the struct tags that declaratively tell what to do.

:)

Also about "hacks": 
- they sent us to the moon and more.
- everything starts with a "hack"
- perfection is the enemy of action
- 🚀

Reply:
Re "hacks" - Completely agreed. Added a comment in the [other issue](https://github.com/pokt-network/pocket/issues/154#issuecomment-1208694291) so we don't forget!